### DISCUSSION TITLE
ow can I read  two TTYs from PyGPSClient  simultaneously?

### DISCUSSION THREAD

**Discussion by ricshsu** [semuconsulting/PyGPSClient#146] [Q&A] [answered]
I have two satellite cards. How can I use PyGPSClient to read satellite data from both simultaneously?"

**Comment by semuadmin** [comment, marked as answer]
Hi @ricshsu,

I'm sorry - you can't read from two GNSS receivers simultaneously using a single instance of PyGPSClient, but you _can_ run more than one instance of PyGPSClient at a time using different inputs.
